apps/console/src/extensions/configs/scim.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {scimConfig} from "./models/scim";
+
+export const scimClaimsMap: scimConfig = {
+
+    oidc: "http://wso2.org/oidc/claim",
+
+    scim: {
+        enterpriseSchema: "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User",
```

Comment:
Possible object key arrangement lint warnings here. Check if lint plugin is configured properly. https://github.com/wso2/identity-apps/blob/master/docs/DEVELOPER.md#eslint

Reply:
Fixed in commit 471d08aaf8bd541cb620cb0af05f4f486ac76c34